docs/content-guide/redirects.md

```diff
@@ -17,6 +17,29 @@ Use redirects for fun and profit!
 
 ---
 
+## Background
+
+If you move an existing article in the documentation, you **must** add redirects. Otherwise, links break between versions and users can't find the information they are looking for, even if it exists in the version they are viewing!
+
+If your product **is not** versioned (e.g., a cloud product), add **one** redirect:
+
+![Diagram showing a single path change and a single redirect](../images/single_redirect_diagram.png)
+
+This redirect ensures any requests to the old URL properly re-route to the new URL. To learn more, refer to [Example redirects](#example-redirects).
+
+If your product **is** versioned, add **three** redirects to reroute requests to the appropriate path in each version:
+
+![Diagram showing three redirects for a versioned product move](../images/three_redirects_per_versioned_move.png)
+
+The three redirects cover the following use cases:
+1. Redirects the latest version URL, which has no version path (i.e., `/product/<path>`).
+2. Redirects older versions to the old path (i.e., `/product/:version/<old_path>`).
+3. Redirects current and future versions to the new path (i.e., `/product/:version/<new_path>`).
+
+Refer to [Versioned redirects](#versioned-redirects) for a full example.
+
+We also recommend that you replace references to the old URL with the new URL in your documentation text.
```

Comment:
This is true regardless of whether your content is versioned. I'd suggest adding this to the very beginning in the vein of: "In addition to changing in-content references when a UR changes blah blah blah"